### ISSUE TITLE
[13.0][BUG][website_event_track]Ribbons do not display on sponsors logo

### ISSUE THREAD

**Issue by jbcrouigneau** [odoo/odoo#44147]
Impacted versions: 13.0

Steps to reproduce:
Create a website
Create an event
Add sponsors
Go to event web page

Current behavior:
Sponsor type just display under the logo (no css)

Expected behavior:
A ribbon should be displayed on the top right corner

Video/Screenshot link (optional):
![Capture d’écran 2020-01-28 à 20 01 45](https://user-images.githubusercontent.com/12152409/73296000-0be64280-4209-11ea-9d85-e9fd05dd5ff8.png)

**Comment by pedrobaeza** [comment]
Hi, thanks for reporting. Are you able to reproduce the problem in runbot infrastructure (http://runbot.odoo.com/runbot - admin/admin)?

If so, please explain with all the possible detailed steps to get to the error. Not having them, it's very difficult to hunt down the problem. Please don't be shy of adding details, and even better if you provide some screenshots or videos.

If not, this can be due to a configuration change, a custom/community module you have deployed in your DB or an outdated code. Please report back if it's the case.

For keeping a clean house in this issue tracker, if you don't answer in several days, this will be closed assuming it was one of the stated things.

**Comment by jbcrouigneau** [comment]
Hello,
No, I can't reproduce on runbot, but the configuration seems very different from mine (much more modules).
I use a fresh 13.0 (13.0-20200127) community edition install (Debian package on Ubuntu 18.04).
I activated only official apps : crm, website, account, sale_management, website_sale, hr, hr_expense, contacts, mail, survey, mass_mailing, calendar, website_blog, website_event
Also added : website_event_sale, attachement_indexation
My Odoo is in French, located in Morocco.
Tried to create a new event exactly the same way as on runbot, still the same issue (see the screenshot).
Not sure if that can help...
